tests/test_messaging.py

```diff
@@ -119,13 +119,17 @@ def test_history(alice, bob, wait_until):
     assert not alice.entry.get()
     alice.previous_message_to_entry()
     assert alice.entry.get() == "first message"
+    alice.next_message_to_entry()
+    assert not alice.entry.get()
+
+    assert not alice.entry.get()
     alice.previous_message_to_entry()
     assert alice.entry.get() == "first message"
-    alice.next_message_to_entry()
+    alice.previous_message_to_entry()
     assert alice.entry.get() == "first message"
```

Comment:
Why does the message stay there when you press down arrow key? This doesn't seem correct to me.

Reply:
It is staying when you press _up_ arrow, not down.

I'm just trying to keep the test logic from master branch:

```

    assert not alice.entry.get()
    alice.previous_message_to_entry()
    assert alice.entry.get() == "first message"
    alice.previous_message_to_entry()
    assert alice.entry.get() == "first message"
    alice.next_message_to_entry()
    assert alice.entry.get() == "first message"
    alice.next_message_to_entry()
```

Here, you press up-arrow to display "first message", then you press up-arrow again, and the content in entry remains the same. Then you press down-arrow. With the logic before, "first message" would still be there. This is what I removed on line 124-127, and I'm keeping the logic of pressing up-arrow twice before pressing down-arrow. However, as you see on the new line 131, it's asserting that it's empty.